package/MDAnalysis/analysis/align.py

```diff
@@ -1350,45 +1350,51 @@ def get_atoms_byres(g, match_mask=np.logical_not(mismatch_mask)):
 
             # stop if we created empty selections (by removing ALL residues...)
             if ag1.n_atoms == 0 or ag2.n_atoms == 0:
-                errmsg = ("Failed to automatically find matching atoms: created empty selections. " +
-                          "Try to improve your selections for mobile and reference.")
+                errmsg = ("Failed to automatically find matching atoms: created empty selections. "
+                           "Try to improve your selections for mobile and reference.")
```

Comment:
one space less. 😉 Maybe we sound fussy but we try to follow the [PEP8](https://www.python.org/dev/peps/pep-0008/#indentation) the best we can.

Reply:
No problem sir, it is always a good practice to maintain a good code flow.